lotman/lotman_test.go

```diff
@@ -267,11 +267,11 @@ func TestUpdateLot(t *testing.T) {
 	require.True(t, success)
 
 	// Update the test-1 lot
-	dedGB := float64(999.0)
+	dedicatedGB := float64(999.0)
```

Comment:
Is there a way to prevent the checker from blocking useful abbreviations like this? I guess it's not a huge issue since most of us tab-complete, but I can see this being annoying in the future if these spell checkers don't let us choose variable names we want because it thinks it knows better...

Reply:
I'm not sure why that specific variable name kept getting flagged despite the following in `_typos.toml`:
```
extend-ignore-identifiers-re = [
    ## Ignore TLAs. Two and three letter acronyms are not worth fixing.
    "^[A-Za-z]{2,3}$",
    ...
```
There were definitely quite a few more names getting flagged before I added that bit of config….

Reply:
Since _typos.toml accounts for most of these (and it's unclear why this one case isn't being filtered), I don't think I'm too worried.